egs2/babel/asr1/setup_experiment.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [ $# -ne 1 ]; then
+  echo >&2 "Usage: ./setup_experiment.sh <expname>"
```

Comment:
We don't need this file?

Reply:
Sure (sorry, it's a mistaken heritage from Kaldi